CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Clarified behavior of Transaction Extension endpoints, notably that PUT or PATCH that changes the `collection`
```

Comment:
It seems these points are not captured in the CHANGELOG:

> Allow partial Item and partial ItemCollection containing partial Items as POST input
> Allow partial Item containing partial Items as PUT input (but the server has to be able to derive a valid Item from it to return from the GET endpoint, e.g., by populating the id and collection fields or adding links)

Reply:
I'll add them